algorithms/follow_marker.py

```diff
@@ -1,17 +1,33 @@
+import interfaces
 import logging
-import core
-import algorithms.heading_hold as hh  # hh to match other styles in our code base
-
-
-def drive_to_marker(speed, drive_board, center, radius, nav_board):
-    angle_to_ball = core.FIELD_OF_VIEW * ((center[0] - core.WIDTH / 2) / core.WIDTH)
-    print(angle_to_ball)
-    if radius == 0:
-        radius = 1  # TODO: more robust fix
-    distance = core.SCALING_FACTOR / radius
-    logging.info("Distance to marker: %f" % distance)
-    logging.info("Angle to marker: %f" % angle_to_ball)
-    goal_heading = nav_board._heading + angle_to_ball
-    left, right = hh.get_motor_power_from_heading(speed, goal_heading, drive_board, nav_board)
-    print(str(left) + "," + str(right))
-    return (left, right), distance
+import algorithms.heading_hold as hh
+
+
+def drive_through_gate():
+    # TODO: Will attempt to drive through a gate composed of two AR Tags
+    pass
+
+
+def drive_to_marker(speed, distance, angle):
+    """
+    Calculates the angle and distance of the AR marker with given center pixels.
+    Then returns drive speeds necessary to stay on course towards it.
+
+    Parameters:
+    -----------
+        speed - the desired drive speed
+        center - the center pixels of the AR Tag detected (x, y)
+
+    Returns:
+    -----------
+        left - the desired speed for left motors
+        right - the desired speed for right motors
+        distance - the distance the AR marker is away from the rover
```

Comment:
While this nice docstring was just added in this PR, the parameters and returns section already appears out of date.